src/constructor.js

```diff
@@ -0,0 +1,32 @@
+class ToDo {
+  constructor() {
+    this.list = localStorage.getItem('toDoTask')
+      ? JSON.parse(localStorage.getItem('toDoTask'))
+      : [];
+  }
```

Comment:
- [ ] The status of the task should be updated when the checkbox is clicked and the status should also be stored in the local storage. Currently, it is set to false even when the checkbox is clicked, kindly modify that.